src/bin/log/tracing_chrome.rs

```diff
@@ -519,14 +519,16 @@ where
         }
     }
 
-    fn get_root_id(&self, span: SpanRef<S>) -> Option<u64> {
+    /// Returns [Option]`<i64>` instead of [Option]`<u64>` because apparently Perfetto gives an
```

Comment:
What is up with the markdown here?
```suggestion
    /// Returns `Option<i64>` instead of `Option<u64>` because apparently Perfetto gives an
```